ci/do_ci.sh

```diff
@@ -148,6 +148,19 @@ elif [[ "$1" == "bazel.dev" ]]; then
   echo "Building and testing..."
   bazel test ${BAZEL_TEST_OPTIONS} -c fastbuild //test/...
   exit 0
+elif [[ "$1" == "bazel.compiletimeoptions" ]]; then
+  # Right now, none of the available compile-time options conflict with each other. If this
+  # changes, this build type may need to be broken up.
+  COMPILE_TIME_OPTIONS="--define=signal_trace=disabled --define hot_restart=disabled --define google_grpc=disabled"
```

Comment:
Add `--define boringssl=fips`? :)

Reply:
Yes, I'll add that as soon as the change is merged.

Reply:
Actually, whichever one is ready to merge first, let's put that in. Then the 2nd one can pickup this change.

Reply:
We need to bump the build image after FIPS is merged.

But yeah, let's not block this on it.

Reply:
Now that it's merged, please add it.